<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="listWidget_contacts">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>256</width>
     <height>341</height>
    </rect>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>116</width>
     <height>26</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>620</y>
     <width>206</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_call">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Call</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hang_up">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Hang Up</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>Connected contacts:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>B  A  B  E  L</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>2 0 1 9</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="box_Pseudo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ConnectMe">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect me</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Pseudo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="box_Friends">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add </string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
